### DISCUSSION TITLE
desinstalar Maxwell’s Aedt

### DISCUSSION THREAD

**Discussion by Herve-L1** [ansys/pyaedt#5046] [Q&A]
How to perform desinstalar Maxwell’s Aedt ?

**Comment by Samuelopez-ansys** [comment]
Hi @Herve-L1 ,

What do you mean? If you want to uninstall Maxwell AEDT, PyAEDT is not the website to search.

If you want to uninstall PyAEDT and you used the PyAEDT installer, then go to your PersonaLib and delete the folder Toolkits. Finally, go to %appdata% directory and delete ./pyadt_env which is the virtual environment with PyAEDT.
